### DISCUSSION TITLE
None of Discord Channel is visible when joining the discord server

### DISCUSSION THREAD

**Discussion by ckt1031** [vercel/turborepo#1677] [Help] [answered]
### What version of Turborepo are you using?

latest

### What package manager are you using / does the bug impact?

npm, pnpm, Yarn v1

### What operating system are you using?

Windows

### Describe the Bug

![image](https://user-images.githubusercontent.com/65409152/183628283-e07c4956-bfb4-4773-b4a5-415002a882c1.png)

### Expected Behavior

![image](https://user-images.githubusercontent.com/65409152/183628371-15b7b3b6-66b4-4204-b651-3cf1f9325a23.png)

All are locked and invisible

### To Reproduce

Join via the link provided.

**Comment by sebastiangug** [comment]
it'd be nice to also be discoverable when searching for 'turborepo' on the explore tab

**Comment by Nutlope** [comment, marked as answer]
Hi @cktsun1031, thanks for reporting! Our Discord bot that handles permissions crashed and we just got it back up so this should be fixed! Please leave the server and re-join, and you should be good.

**Comment by Palanikannan1437** [comment]
I get the same error @Nutlope, am I missing something here 😅?

<img width="1350" alt="image" src="https://github.com/vercel/turbo/assets/73993394/bd6a67dc-e138-4204-a03b-2f4c799e767b">

**Comment by Bejasc** [comment]
I'm also having the same issue.
